src/main/java/dinkplugin/notifiers/LevelNotifier.java

```diff
@@ -155,7 +154,7 @@ private void handleLevelUp(Skill skill, int level, int xp) {
         int xpInterval = config.xpInterval() * 1_000_000;
         if (enabled && xpInterval > 0 && level >= MAX_REAL_LEVEL && xp > previousXp) {
             int remainder = xp % xpInterval;
-            if (remainder == 0 || xp - remainder > previousXp) {
+            if (remainder == 0 || xp - remainder > previousXp || (xp >= Experience.MAX_SKILL_XP && !config.levelNotifyVirtual())) {
```

Comment:
Maybe just fire regardless of levelNotifyVirtual & wait for someone to complain about duplicate events at 200m?
For Discord use cases, this might be super mini annoying, but for custom JSON events you might as well fire both imo